### PR TITLE
feat: added supported apps on homepage

### PR DESCRIPTION
**Pull request by ryanadhi**
Closes https://github.com/kawalcovid19/wargabantuwarga.com/issues/710

## Description

Adding supported apps card
Screenshot 2021-08-14 223902

## Current Tasks

- [x] added card
- [x] added testing for new card

### CODE REVIEW

File: _content/home-page.md [hunk 1]
```diff
@@ -2,10 +2,7 @@
 last_updated_time: 2021-08-09T02:58:39.509Z
 home_banner_image: https://res.cloudinary.com/wargabantuwarga/image/upload/v1627049958/hero_banner_desktop_zat71c.png
 ---
-#### Mau Konsultasi Medis dengan Dokter (Telemedicine)?
```
Comment (mazipan) [inline _content/home-page.md:5]: You don't need to remove this part via codebase.
We can remove later from CMS.
Just focus on the new section.

Comment (ryanadhi) [reply _content/home-page.md:5]: noted. just pushed the restored version from this file

**Comment by ryanadhi** [comment]
@zainfathoni and @mazipan should I make this draft as ready or should I wait for https://github.com/kawalcovid19/wargabantuwarga.com/pull/706 ?

**Comment by mazipan** [comment]
block by #706

**Comment by resir014** [comment]
@ryanadhi #706 has been merged. You can resume work on this PR now.

**Comment by ryanadhi** [comment]
@resir014 I just updated my PR and made it ready to be reviewed. Thanks

File: components/home/homepage-supported-apps.tsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+      <h2 className="font-semibold">
+        Download aplikasi telemedicine yang
+        <br />
+        didukung Kemenkes:
+      </h2>
+      <ul className="grid grid-cols-2 gap-2 sm:gap-4 sm:grid-cols-4">
+        {supportedApps.application.map((app, index) => (
+          <li
+            key={index}
+            className="inline-flex flex-col items-center justify-center text-center relative  py-3 px-2"
```
Comment (resir014) [inline components/home/homepage-supported-apps.tsx:19]:
```suggestion
            className="inline-flex flex-col items-center justify-center text-center relative"
```

We can remove these padding to bring everything closer together.

**Review by resir014 — CHANGES_REQUESTED** [review]

**Comment by resir014** [comment]
@ryanadhi This is good, just one minor change needed. Please look into them, thanks!

File: components/home/homepage-supported-apps.tsx (hunk 2 above)
Comment (ryanadhi) [reply components/home/homepage-supported-apps.tsx]: I just pushed new commit without padding. thanks

**Review by resir014 — APPROVED** [review]

**Pull request merged**